packages/fiori/src/ShellBar.hbs

```diff
@@ -41,9 +47,7 @@
 
 
 						{{#if showTitleInMenuButton}}
-							<h1 class="ui5-shellbar-menu-button-title">
```

Comment:
Some of the required styles are lost if this class is removed. Can we change the H1 to DIV here and keep the `ui5-shellbar-menu-button-title` class?